### ISSUE TITLE
Mist UI - only 'Tests' section visible

### ISSUE THREAD

**Issue by JorreB** [ethereum/mist#2202]
#### System information
``````
Version: `0.8.9`
OS & Version: Windows 10
Node type: `eth/geth(default)` 
``````

I can only view the 'Tests' section of Mist.  As a result, I cannot see my accounts.  Mist starts and appears to synch normally. The accounts still exist in the backup folder.   When I use the old 'Wallet' app, it works as normal and I can see the accounts.

I recently accepted the download of geth 1.6 when I started Mist.  I did not accept the download for the Wallet, and it appears to still be running on 1.59.  Other than that, I cannot think of anything else I have changed. 

How can I view my accounts again in Mist?

- Screenshots
![image](https://cloud.githubusercontent.com/assets/28385509/25695346/9f9fb1c8-30aa-11e7-9d07-a56ec81711f1.png)

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot
![image](https://cloud.githubusercontent.com/assets/28385509/25695528/64be730e-30ab-11e7-96c8-1ea08db2db9e.png)

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by JorreB** [comment]
I fixed this by downloading a previous version of Mist.  When I opened it, it appeared briefly to have only the 'Test' tab, before quickly reverting to a normal state. I assume it was able to fix some .ini file which had been corrupted.

**Comment by evertonfraga** [comment]
Hello, this should be fixed at Mist 0.8.10 version, just released.

The problem was about concurrency on saving the internal database that accounts for all tabs. More info here: https://github.com/ethereum/mist/pull/2125

I'm closing this for now. Thanks for reporting.
